src/navigator/enums/modals.enum.ts

```diff
@@ -32,5 +36,14 @@ export type ModalsParamList = {
   [ModalsEnum.ImportAccount]: undefined;
   [ModalsEnum.CollectibleModal]: { collectible: TokenInterface };
   [ModalsEnum.AddCustomRpc]: undefined;
-  [ModalsEnum.RemoveLiquidity]: undefined;
+  [ModalsEnum.RemoveLiquidity]: {
+    lpContract: ContractInterface<LiquidityBakingContractAbstraction | undefined, LiquidityBakingStorage>;
+    aToken: TokenInterface;
+    bToken: TokenInterface;
+  };
+  [ModalsEnum.AddLiquidity]: {
+    lpContract: ContractInterface<LiquidityBakingContractAbstraction | undefined, LiquidityBakingStorage>;
```

Comment:
update `ContractInterface`  field  `contract: C | undefined;` instead of passing `| undefined` everywhere